src/app/components/categories/categories.component.html

```diff
@@ -0,0 +1,15 @@
+<section id="categories">
+  <div class="categories-category" *ngFor="let category of categories">
+    <a class="categories-header" [id]="category.name + '-header'" routerLink="/categories/{{category.id}}">
+      <h1 [id]="category.name + '-title'">{{category.name}}</h1>
+      <p routerLink="/categories/{{category.id}}">Voir plus</p>
+      <p><strong>></strong></p>
+    </a>
+
+    <div class="categories-3articles">
+      <article></article>
```

Comment:
?

Reply:
empty articles for the precedent sprint to show how it would have been organised
Will be modified this sprint